src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/loadbalancer/ApacheEc2LoadBalancerFormula.kt

```diff
@@ -82,25 +86,48 @@ internal class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
     }
 }
 
-private class ApacheLoadBalancer(
+private class HaProxyLoadBalancer(
     private val nodes: List<URI>,
     private val ssh: Ssh,
     private val httpPort: Int
 ) : LoadBalancer {
 
-    init {
-        throw Exception("Wait until the `infrastructure` module ships `ApacheLoadBalancer`")
-    }
-
     override val uri: URI = URI("http://${ssh.host.ipAddress}:$httpPort/")
 
     override fun waitUntilHealthy(
         timeout: Duration
     ) {
-        throw Exception(
```

Comment:
This impl is empty. Is the `uri` available immediately?

Reply:
It will available immediately because provision method is restarting apache2 service at the end and the call is blocking.